apps/server/src/shared/connect/contract.ts

```diff
@@ -50,40 +52,40 @@ export interface Member {
   //     And you really should be doing this thing :) Thank you!!! <a href=\https://google.com\ id=\instructional_text\>My Test Link</a>
   //   ]
   // }
```

Comment:
Can the above commented code be removed?

Reply:
That would have to be done in ucw-app. I think all the changes in this file came from pulling there.

Reply:
Oh right.